duckduckgo/Android
# Remove unnecessary stub tests

Task/Issue URL: https://app.asana.com/0/488551667048375/1200036862712904/f
Tech Design URL: 
CC: 

**Description**:
Deleting stub tests which are no longer necessary given changes made to CI server

**Steps to test this PR**:
1. Ensure CI passes
2. Ensure you can see test reports for all the instrumentation test results


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
